parts/k8s/addons/secrets-store-csi-driver.yaml

```diff
@@ -451,16 +458,26 @@ spec:
         app: csi-secrets-store-provider-azure
     spec:
       serviceAccountName: csi-secrets-store-provider-azure
+      hostNetwork: true
```

Comment:
So `0.0.9` for provider is not compatible with `v0.0.14` release of the driver. The following flag `--grpc-supported-providers=azure` needs to be added to the `secrets-store` container for the driver-provider communication to work. 

Ref doc - 
https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/docs/how-to-upgrade.md

`--grpc-supported-providers=azure` needs to be added in [here](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/deploy/secrets-store-csi-driver.yaml#L53-L58)

Reply:
FYI, for users using helm chart to deploy the driver and provider we configure it automatically - https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/charts/csi-secrets-store-provider-azure/values.yaml#L75